Remotion/Validation/Core/RuleCollectors/DeferredInitializationValidationMessage.cs

```diff
@@ -14,6 +15,8 @@ public DeferredInitializationValidationMessage ()
 
     public void Initialize (ValidationMessage validationMessage)
     {
+      ArgumentUtility.CheckNotNull("validationMessage", validationMessage);
```

Comment:
There is also the option to use `nameof(validationMessage)` which would be beneficial in the case of wanting to rename the parameter. I haven't seen it used that much in re-motion though, but it might be useful to know. 

@MichaelKetting is there a reason for not using `nameof`?